### DISCUSSION TITLE
SUGGESTION: User option may consolidate threads on timeline

### DISCUSSION THREAD

**Discussion by AirsoftDad** [mastodon/mastodon#23357] [General]
If this features exists, please let me know.  I want a way to show all threads in a grouped manner instead of having a running dialogue for that thread on my timeline.  If someone replies to that Toot, that thread will go to the top of my timeline unless I choose to follow that thread no longer. The idea is to organize threads in the same manner I can organize my email inbox, should I choose.

Again, if this feature exists, please let me know. If not, I would like to suggest such a feature for you to consider in the future.

**Comment by StyXman** [comment]
For comparison, this sounds like mutt's and thunderbird's threading models (one of them, any ways; both have at least a couple). I think reddit and /. also implement this.
